services/second-service/src/posts/posts.controller.ts

```diff
@@ -23,15 +20,15 @@ import { GetPostsDto, PostsDto } from './dto/posts.dto';
 import { ApiBody, ApiTags } from '@nestjs/swagger';
 import { swaggerConfig } from '../swagger.config';
 
-import { TransformPaginationResponseInterceptor } from '../common/interceptors';
+import { TransforPaginationResponse } from '../common/pagination/transform-pagination-response.decorator';
 
 @ApiTags('Posts')
 @Controller('posts')
 export default class PostsController {
   constructor(private readonly postsService: PostsService) { }
 
   @Get()
-  @UseInterceptors(new TransformPaginationResponseInterceptor(PostsDto, { excludeExtraneousValues: true }))
+  @TransforPaginationResponse(PostsDto, { excludeExtraneousValues: true })
```

Comment:
Since we use `excludeExtraneousValues` everywhere, would it make sense to default it to `true`?

Reply:
Still would like to keep this configurable

Reply:
That's why I suggested have a default `true`, not remove completely